### PR TITLE
News 2.0.0

### PR DESCRIPTION
**Pull request by ajeddeloh**
Depends on https://github.com/coreos/ignition/pull/772

### CODE REVIEW

**Comment by ajeddeloh** [comment]
Typically we did spec change info in migrating configs doc, which I have yet to write (working on that next, along with 8000 other things to polish this)

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+
+  NOTE: This is an alpha release. While the spec is marked as stable (i.e no
+    "-experimental" suffix) we still reserve the right to change it until the
+    stable 2.0.0 release. However, we do not anticipate any backwards
+    incompatible changes aside from removing deprecated fields.
+
+  NOTE: In order to allow types from both the 2.x.y and 3.0.0 specs to be
+    vendored and imported in the same project, we are skipping version 1.0.0.
+    Go mod (and some other tools) treat v0.x and v1.x as the same when
+    importing packages with semantic import versioning)
```
Comment (bgilbert) [inline NEWS]: `)`

**Comment by ajeddeloh** [comment]
Fixed extra close paren

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+
+  Features:
+
+    - Ignition now understands config specification 3.0.0
+    - Configs are now merged instead of appended
+
+  Changes:
+
+    - Configs with version < 3.0.0 are now rejected
+    - Duplicate entries are now disallowed in lists
```
Comment (arithx) [inline NEWS]: Could be pointing out that it's only in some lists (i.e. you can have duplicates in appended contents). Not a deal breaker either way.

**Pull request merged**